### PR TITLE
Add solution to SENG bridge building activity

### PR DESCRIPTION
**Pull request by eAlasdair**
Adds the following sketch of a potential solution to `Fun with the Waterfall and Agile processes` activity in the Software Engineering section. Also links to [here](https://github.com/uccser/cs-field-guide/issues/505#issuecomment-494605977) for an image of the solution in action - it's too low of a production value to include in the content itself. Alternatively I could link to [the picture only](https://user-images.githubusercontent.com/25916475/58138739-b88c1f80-7c8b-11e9-9195-b25eabbc725a.jpg).

Addresses (does not resolve) #505 

image

### CODE REVIEW

**Comment by eAlasdair** [comment]
Here is a better image, I'll crop the top out
IMG_4882

**Comment by courtneycb** [comment]
> Where are we going to host it - in the content or somewhere else?

Maybe in static > img > chapters?

**Comment by eAlasdair** [comment]
> > Where are we going to host it - in the content or somewhere else?
> 
> Maybe in static > img > chapters?

And then just put the image under the diagram?

**Comment by courtneycb** [comment]
> Where are we going to host it - in the content or somewhere else?

Thoughts on just having the image in the content instead of the uninteractive?

**Comment by courtneycb** [comment]
> And then just put the image under the diagram?

Lol soz I deleted my comment cause I realised what you were actually meaning

**Comment by eAlasdair** [comment]
Can't test it yet, but the image has been replaced

File: csfieldguide/chapters/content/en/software-engineering/sections/software-processes.md [hunk 1]
```diff
@@ -152,6 +152,16 @@ Which of these two approaches worked best &ndash; designing everything first, or
 
 {panel type="teacher-note"}
 
+# Solution
+
+Here is an example solution students might come up with after a few revisions:
+
+{image file-path="img/chapters/card-bridge-solution.JPG" alt="Image of a potential solution to the card bridge"}
```
Comment (courtneycb) [inline csfieldguide/chapters/content/en/software-engineering/sections/software-processes.md]: insanely picky but can we have the jpeg extension lowercase? :joy:

**Pull request merged**